packages/babel-preset-gatsby/src/index.js

```diff
@@ -77,8 +77,10 @@ export default function preset(_, options = {}) {
         resolve(`@babel/preset-react`),
         {
           useBuiltIns: true,
-          pragma: `React.createElement`,
+          pragma:
+            options.runtime === `automatic` ? null : `React.createElement`,
```

Comment:
Interesting that they don't set this to null themselves. Are you sure we need this? Would undefined be better here?

Reply:
Yeah. If I don't do this change the plugin throws this error `pragma and pragmaFrag cannot be set when runtime is automatic.`

I'll test `undefined` and use that if I can.